src/lib/messages.js

```diff
@@ -0,0 +1,91 @@
+import { storage } from 'sdk/simple-storage';
+import tabs from 'sdk/tabs';
+import { getMostRecentBrowserWindow } from 'sdk/window/utils';
+
+import measure from '../measurements';
+import { e10sProcessCount, e10sStatus } from './e10s-status';
+import Logger from './log';
+import sendEvent from './metrics';
+
+const logger = new Logger(
+  'sdk.lib.messages',
+  getMostRecentBrowserWindow().console
+);
+
+export const storeTab = uuid => {
```

Comment:
What's the point of using an arrow function at the top level of a module, instead of just the function form? There's no need for the arrow function 'this' behavior and the function form is both shorter and way less ugly.